api/v1/app.py

```diff
@@ -37,3 +36,6 @@ def close_connection(exception):
 
 if __name__ == '__main__':
    app.run(host="0.0.0.0", port=5000, threaded=True)
+   host = getenv("HBNB_API_HOST", "0.0.0.0")
+   port = getenv("HBNB_API_PORT", "5000")
```

Comment:
Those variables won't work after app.run(). The main idea using environment variables is to avoid hardcoded values. You should get the host and port values before app.run() and call those variables as parameters on app.run()

Reply:
the change was made it. Ready for merge the branch